docs/internal/api-spec.swagger.yml

```diff
@@ -630,15 +620,17 @@ definitions:
     type: object
     required:
       - type
-      - transaction_id
-      - position
+      - output_id
```

Comment:
In the `description` field for this object, it would be nice to indicate that `output_id` is the preferred method of referencing the prevout, but the pair of `transaction_id` and `position` are supported but deprecated.